scripts/doneTask.js

```diff
@@ -0,0 +1,11 @@
+import { renderTodoList } from "../index.js";
+import { todos } from "./deteleTask.js";
+
+export const handleDone = (taskId, doneState) => {
+  const task = todos.find((todo) => todo.id === taskId);
+  console.log({ doneState });
+  if (doneState) {
+    task.isDone = true;
+  } else task.isDone = false;
```

Comment:
This code can be a oneliner if we try. Notice the use of `Boolean`, which indicates `isDone` can also be of other types and we'll coerce it to a `boolean` type.
```suggestion
  task.IsDone = Boolean(doneState)
```